### PR TITLE
[test] Update convolution

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: torch_xla/csrc/convolution.cpp [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #include "torch_xla/csrc/convolution.h"
 
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
 #include "tensorflow/compiler/tf2xla/kernels/conv_op_helpers.h"
```
Comment (miladm) [inline torch_xla/csrc/convolution.cpp:5]: should this line stay?

Comment (ManfeiBai) [reply torch_xla/csrc/convolution.cpp]: Thanks, Milad, this line not needed and deleted in the newer commit

**Comment by JackCaoG** [comment]
`test_Conv2d_naive_groups_xla_float32` seems to be a real error.

**Comment by ManfeiBai** [comment]
checked again, funcs in tf has changed a lot, so also have a new PR with the newest funcs: https://github.com/pytorch/xla/pull/5178

**Comment by ManfeiBai** [comment]
`test_Conv2d_naive_groups_xla_float32 ` failed because our code is very old, please track in https://github.com/pytorch/xla/pull/5185

**Pull request closed without merging**